frontend/src/components/RestaurantDetails/RestaurantInfo.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import {withStyles} from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import Divider from '@material-ui/core/Divider';
+
+import CardMedia from '@material-ui/core/CardMedia';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import {Link} from "react-router-dom";
+
+const styles = {
+    card: {
+        width: 1200,
+        background: 'rgba(230, 232, 209, 0.96)',
+        marginTop: 7,
```

Comment:
According material design you need use 8px step. So use value divided by 8 or pass `theme` Object to your styles, and use `theme.spacing.unit` multiple by some value